patterns/typescript/src/deterministic_idempotency_tokens.ts

```diff
@@ -0,0 +1,102 @@
+import * as restate from "@restatedev/restate-sdk";
+import { randomUUID } from "crypto";
```

Comment:
Isn't it more common to use `uuidv4` instead of `crypto`?

Reply:
maybe, I really don't know. I often use `crypto` because it is built-in, so no dependency needed. `uuidv4` needs to install and package additional dependencies.